rasa/core/trackers.py

```diff
@@ -695,12 +722,41 @@ def _merge_slots(
         ]
         return new_slots
 
+    # pytype: disable=bad-return-type
+    @property
     def active_loop_name(self) -> Optional[Text]:
         """Get the name of the currently active loop.
 
         Returns: `None` if no active loop or the name of the currently active loop.
         """
-        if not self.active_loop:
+        if not self.active_loop or self.active_loop.get(LOOP_NAME) == SHOULD_NOT_BE_SET:
             return None
 
-        return self.active_loop.get("name")
+        return self.active_loop.get(LOOP_NAME)
+
+    # pytype: enable=bad-return-type
+
```

Comment:
```suggestion
```

Reply:
I'd keep the line since enable is referred to previous method